release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -438,6 +438,16 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 [id="ocp-4-17-networking_{context}"]
 === Networking
 
+[id="ocp-4-17-support-for-bgp-routing-protocol_{context}"]
```

Comment:
Wrong file. This should be a 4.18 GA release note, not a 4.17 GA note.

Reply:
@cgoncalves, yep, I opened this against 4.17 when this was a potential 4.17 feature. I haven't had time to rebase it against the 4.18 branch.